packages/create-nuxt-app/lib/saofile.js

```diff
@@ -76,6 +76,18 @@ module.exports = {
       })
     }
 
+    const eslint = this.answers.linter.includes('eslint')
+    const prettier = this.answers.linter.includes('prettier')
+    const lintStaged = eslint && this.answers.linter.includes('lintStaged')
+    const stylelint = this.answers.linter.includes('stylelint')
+    if (this.answers.ci !== 'none' && (this.answers.test !== 'none' || eslint || prettier || lintStaged || stylelint)) {
```

Comment:
I think this can be sth like `this.answers.linter.length > 0`

Reply:
initially I did do that but the tests were failing, can you please try and reproduce it at your end?

I've added you as collaborator to https://github.com/vinayakkulkarni/create-nuxt-app

Reply:
NP, I’ll try it today

Reply:
@vinayakkulkarni Updated, can you please review ?

Reply:
LGTM